draft-ietf-quic-transport.md

```diff
@@ -2893,9 +2893,12 @@ difficult to distinguish from other packets if the recipient requires the use of
 a connection ID.  To prevent a resulting stateless reset from being trivially
 distinguishable from a valid packet, all packets sent by an endpoint SHOULD be
 padded to at least 22 bytes longer than the minimum connection ID that the
-endpoint might use.  An endpoint that sends a stateless reset in response to a
-packet that is 43 bytes or less in length SHOULD send a stateless reset that is
-one byte shorter than the packet it responds to.
+endpoint might request that a peer include in the Destination Connection ID
+field.  This ensures that any stateless reset sent by the peer is
+indistinguishable from a valid packet sent to the endpoint.  An endpoint that
```

Comment:
This is a bit repetitive to the text 4 paragraphs up, but I'm ok with that if it helps.